obspy/io/reftek/tests/test_core.py

```diff
@@ -0,0 +1,390 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA @UnusedWildImport
+
+import inspect
+import os
+import re
+import unittest
+import warnings
+
+import numpy as np
+
+import obspy
+from obspy.core.util import NamedTemporaryFile
+from obspy.io.reftek.core import _read_reftek130, _is_reftek130, Reftek130
+
+
+class ReftekTestCase(unittest.TestCase):
+    """
+    Test suite for obspy.io.reftek
+    """
+    def setUp(self):
+        try:
+            # doctests of __init__.py produce warnings that get caught. if we
+            # don't raze the slate out the registry here, we can't test those
+            # warnings in the unit tests (if doctests run before unittests)..
```

Comment:
Which warnings do you mean? If you're referring to line 381; that block is missing the `warnings.simplefilter('always')` that is in every other one.